extensions/wikia/Wall/WallExternalController.class.php

```diff
@@ -264,10 +264,20 @@ public function postNewMessage() {
 		}
 
 		$title = Title::newFromText( $this->request->getVal( 'pagetitle' ), $ns );
-		$wallMessage = WallMessage::buildNewMessageAndPost( $body, $title, $this->wg->User, $titleMeta, false, $relatedTopics, true, $notifyEveryone );
 
-		if ( $wallMessage === false ) {
-			error_log( 'WALL_NOAC_ON_POST' );
+		try {
+			$builder =
```

Comment:
I don't have strong opinion about this but what about removing the new line after = ? (here and in other places)